### PR TITLE
60. | Permutation Sequence | C++ | Resolves #931

### LINKED ISSUE #931: 60. | Permutation Sequence | C++
Problem name: Permutation Sequence
Difficulty: Hard
Language: C++

### PR DESCRIPTION
**Pull request by agrawal-deeksha**
Resolves #931

### CODE REVIEW

**Comment by agrawal-deeksha** [comment]
Resolves #931

**Comment by noisefilter19** [comment]
Why are there 2 files in the PR?

File: C++/Valid_Number.cpp [hunk 1]
```diff
@@ -0,0 +1,63 @@
+//https://leetcode.com/problems/valid-number
```
Comment (noisefilter19) [inline C++/Valid_Number.cpp:1]: Please delete this file

**Review by noisefilter19 — CHANGES_REQUESTED** [review]

File: C++/Valid_Number.cpp (hunk 1 above)
Comment (agrawal-deeksha) [reply C++/Valid_Number.cpp]: Deleted.

**Review by noisefilter19 — APPROVED** [review]

**Pull request merged**